docs/handbook/getting-started.md

```diff
@@ -582,13 +580,13 @@ Replace `X` with the appropriate letter for your USB device.
 #### Method 1 (New releases)
 
 ```
-sudo dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
+sudo /usr/gnu/bin/dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
```

Comment:
What is /dev/sdb in illumos? It should have been /dev/rdsk/cXtYd0p0 , wjere X and Y are 0,1 and so on.